docs/content/getting-started/getting-help.mdx

```diff
@@ -1,84 +1,197 @@
+        <a href="#tips">Tips section</a> for tips on including code snippets and
+        how to ask questions that get answered.
+      </td>
+    </tr>
+  </tbody>
+</table>
 
-Search engines like Google generally do a good job at surfacing relevant Dagster docs, Github Discussions and Github Issues, so we recommend starting there.
-
-Unfortunately, ChatGPT is not currently a great resource for answering questions about Dagster, because the corpus it was trained on doesn’t include Dagster’s latest APIs.
+---
 
-However, one can interact with a fine-tuned Language Learning Model (LLM) trained on the latest Dagster docs, Github Discussions and Github Issues right here within our docs pages! Simply click the **Ask AI** button on any docs page to open a chat window to it.
+## Tips
 
-Our LLM is also available in the [#ask-ai](https://dagster.slack.com/archives/C066HKS7EG1) channel of the Dagster Slack.
+### Code snippets
 
----
+#### Use Markdown to create code blocks
 
-## Bugs and feature requests
+When posting in GitHub or Slack, including a code snippet can help others understand your question. Using Markdown to format the content as code improves readability. For example:
 
-We use [Github Issues](https://github.com/dagster-io/dagster/issues) to track all bugs and feature requests from the community, and we welcome issues submitted by users.
-
-If you find a Github issue that seems like it’s related to the issue you’re experiencing, but you're not sure, don’t be shy about posting on that Github issue to ask. We can redirect it to a different issue if necessary.
+      ```   # backticks create a code block
+      from dagster import ...
+      ...
+      ```
 
-### Tips for filing issues
+#### Add syntax highlighting
 
-- For bugs, include the minimum-sized working code snippet that reproduces your issue.
-- If your Github issue includes a code snippet, you can add syntax highlighting by specifying the language at the top of the block:
+On GitHub - whether in issues or discussions - you can add syntax highlighting to code snippets to further improve readability. This is done by specifying the language at the top of the Markdown code block:
 
-      ```python
+      ```python    ## specifies language
       from dagster import ...
       ...
       ```
 
----
+### Bug reports
 
-## Questions
+When filing a [GitHub issue](https://github.com/dagster-io/dagster/issues) for a bug:
 
-If you're trying to find out the best way to use Dagster for your use case, there are a couple places to get help.
+- **DO** include the minimum-sized working code snippet that reproduces the issue
+- **DO** include the steps you took to create the issue, if the issue occurred in the Dagster UI
+- **DO** include the Dagster version being used, if applicable
+- **DON'T** include any sensitive information
 
-### Github Discussions: the preferred place to ask questions
+### Ask questions that get answers
```

Comment:
actually, do we want to promote this to the top? i feel this is actually the most important tip.

Reply:
no i dont feel strongly. your call!